src/libs/actions/PersonalDetails.js

```diff
@@ -171,10 +171,14 @@ function getFromReportParticipants(reports) {
             Onyx.merge(ONYXKEYS.PERSONAL_DETAILS, formattedPersonalDetails);
 
             // The personalDetails of the participants contain their avatar images. Here we'll go over each
-            // report and based on the participants we'll link up their avatars to report icons.
+            // report and based on the participants we'll link up their avatars to report icons. This will
+            // skip over default rooms which aren't named by participants.
             const reportsToUpdate = {};
+            const defaultRooms = _.values(CONST.REPORT.CHAT_TYPE.DEFAULT_ROOMS);
             _.each(reports, (report) => {
-                if (report.participants.length > 0) {
+                if (_.contains(defaultRooms, report.chatType)) {
+                    reportsToUpdate[`${ONYXKEYS.COLLECTION.REPORT}${report.reportID}`] = {icons: [getDefaultAvatar()]};
```

Comment:
What about the `reportName`? Also, can you work with @Expensify/design to get that lounge chair icon and add it to the app?

Reply:
I think report name doesn't need to change here because we don't need to change it for defaultRooms right? They're already being set in `getChatReportName`. But this part will look a little different, it might make more sense in the new version.